CHANGELOG.md

```diff
@@ -3,6 +3,177 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v6.0.0-alpha.3
+
+_Oct 13, 2022_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Simplify `useGridApiContext` and `useGridApiRef` type overrides (#6423) @cherniavskii
```

Comment:
We clearly need a highlight for the new fields

@joserodolfofreitas @LukasTy @alexfauquette 

First draft:

- ⌚️ New components to edit dates and times with the beyboard, without using any modal or dropdown UI.
  Please check out our [documentation](https://next.mui.com/x/react-date-pickers/fields/) to discover those new components.

  - [`DateField`](https://next.mui.com/x/react-date-pickers/date-field/) to edit a single date
  - [`TimeField`](https://next.mui.com/x/react-date-pickers/time-field/)  to edit a single time
  - [`DateTimeField`](https://next.mui.com/x/react-date-pickers/date-time-field/)  to edit a single date and time
  - [`MultiInputDateRangeField` / `SingleInputDateRangeField`](https://next.mui.com/x/react-date-pickers/date-range-field/) to edit a date range with two inputs
  - [`MultiInputTimeRangeField`](https://next.mui.com/x/react-date-pickers/time-range-field/) to edit a time range with two inputs
  - [`MultiInputDateTimeRangeField`](https://next.mui.com/x/react-date-pickers/date-time-range-field/) to edit a date time range with two inputs
  
  ⚠️ These components are unstable.
  They might receive breaking changes on their props to have the best components possible by the time of the stable release.

Reply:
Minor tweaking:

- ⌚️ New components to edit date and time with <kbd>keyboard</kbd>—without using any modal or dropdown UI.
  Please check out our [documentation](https://next.mui.com/x/react-date-pickers/fields/) to discover those new components.

  - [`DateField`](https://next.mui.com/x/react-date-pickers/date-field/) to edit date
  - [`TimeField`](https://next.mui.com/x/react-date-pickers/time-field/)  to edit time
  - [`DateTimeField`](https://next.mui.com/x/react-date-pickers/date-time-field/)  to edit date and time
  - [`MultiInputDateRangeField` / `SingleInputDateRangeField`](https://next.mui.com/x/react-date-pickers/date-range-field/) to edit date range
  - [`MultiInputTimeRangeField`](https://next.mui.com/x/react-date-pickers/time-range-field/) to edit time range with two inputs
  - [`MultiInputDateTimeRangeField`](https://next.mui.com/x/react-date-pickers/date-time-range-field/) to edit date and time range with two inputs
  
  ⚠️ These components are unstable.
  They might receive breaking changes on their props to have the best components possible by the time of the stable release.

Reply:
Added